raiden/network/transport/matrix/transport.py

```diff
@@ -362,14 +367,17 @@ def start(  # type: ignore
         else:
             prev_user_id = prev_access_token = None
 
+        self._address_mgr.start()
```

Comment:
the address manager wasn't used before?

Reply:
Yes it was, but it didn't have an explicit lifecycle before.